workers/unity/ProjectSettings/EditorBuildSettings.asset

```diff
@@ -5,13 +5,13 @@ EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
   m_Scenes:
-  - enabled: 1
+  - enabled: 0
```

Comment:
We probably don't want to commit this part, right?

Reply:
reverting